admin_pages/transactions/Transactions_Admin_Page.core.php

```diff
@@ -1062,26 +1043,22 @@ public function txn_details_meta_box()
         $this->_template_args['line_item_table'] = $Line_Item_Display->display_line_item(
             $this->_transaction->total_line_item()
         );
-        $this->_template_args['REG_code'] = $this->_transaction->get_first_related('Registration')
-                                                               ->get('REG_code');
+        $this->_template_args['REG_code'] = $this->_transaction->primary_registration()->reg_code();
```

Comment:
Are you certain that the `primary_registration()` will always return the same registration as what `get_first_related` would?  On the surface it looks like you're changing the logic here (`EE_Transaction:primary_registration()` does more work under the hood).

Reply:
> Are you certain that the primary_registration() will always return the same registration as what get_first_related would?

no I'm not certain at all, which is why I changed it, because what we want IS the primary registrant, not just whatever one gets returned first. The `REG_code` template argument is used as a default value for the `Notes / Extra Accounting` input when you apply a payment to the transaction. Here's the help text from that input:
```
An extra field you may use for accounting purposes or simple notes. Defaults to the primary registrant's registration code.
```

Reply:
As stated, this is only being used to populate a **template arg** which is only being used in one place.
This is not the same thing as a regular model object getter where "there may be code working with the existing behaviour"